pyglmnet/pyglmnet.py

```diff
@@ -445,11 +462,21 @@ def _gradhess_logloss_1d(self, xk, y, z):
             gk = np.sum((mu - y) * xk)
             hk = np.sum(mu * (1.0 - mu) * xk * xk)
 
+        elif self.distr == 'probit':
+            prob = probit(z)
+            grad_prob = probit(z)
+            gk = np.sum(y * (grad_prob / prob) -
+                        (1 - y) * (grad_prob / (1 - prob)) * xk)
+
+            mid_l = y * (z * prob + grad_prob) / (prob ** 2)
+            mid_r = (1 - y) * (-z * (1 - prob) + grad_prob) / ((1 - prob) ** 2)
+            hk = np.sum(prob * (mid_l + mid_r) * (xk * xk))
```

Comment:
`np.sum(x * y)` will be slow ... we should use vectorized operations wherever possible